packages/cfengine/build.sh

```diff
@@ -2,7 +2,8 @@ TERMUX_PKG_HOMEPAGE=https://cfengine.com/
 TERMUX_PKG_DESCRIPTION="CFEngine is a configuration management technology."
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@cfengine"
-TERMUX_PKG_VERSION=1:3.17.0
+TERMUX_PKG_VERSION=2:3.17.0
```

Comment:
@xeffyr is this correct? bump the epoch +1 and do I need to specify a TERMUX_PKG_REVISION?

Thanks!

Reply:
No need to bump epoch. You need only to set revision.

Reply:
Well. I was thinking I would need to bump epoch version here since the version hasn't changed?

Are you saying that NOT changing the epoch will cause this new 1:3.17.0 version to overwrite and upgrade to the newer version?

Reply:
Yes, I'm saying to leave epoch as-is and just set `TERMUX_PKG_REVISION`.